cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -9,10 +9,16 @@ KUBECONFIG_PATH="${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig"
 
 MASTER_NODE="${CLUSTER_NAME}-control-plane"
 WORKER_NODE_ROOT="${CLUSTER_NAME}-worker"
+NUM_PFS_REQUIRED=${NUM_PFS_REQUIRED:-1}
```

Comment:
I would change this variable name to `PFS_COUNT_PER_NODE` 
so it will reflect better what it affects

Reply:
It allocates only for the worker not to the master, so should i use `PFS_COUNT_PER_NODE` or 
`PFS_COUNT_PER_WORKER_NODE` ?

Reply:
I would go for the first one I dont think we need to specify the role of the node
I added COUNT to indicate it is expected to be a number

Reply:
Done

Reply:
I vote for `PF_COUNT_PER_NODE`.
You could use "PFS_PER_NODE", but then that plural looks strange. Personally I do not mind, both seem to be expressive enough.